### PR TITLE
Update periodic jobs to point to release-0.20

### PR DESCRIPTION
**Pull request by rahulbabu95**
*Description of changes:*
Update periodic jobs to point to release-0.20

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: jobs/aws/eks-anywhere-build-tooling/eks-anywhere-attribution-periodics-release-0.20.yaml
Comment (sp1999) [inline jobs/aws/eks-anywhere-build-tooling/eks-anywhere-attribution-periodics-release-0.20.yaml:1]: You probably need to update this file as well:
https://github.com/aws/eks-anywhere-prow-jobs/blob/main/templater/jobs/periodic/eks-anywhere-build-tooling/eks-anywhere-attribution-periodics-release-0.19.yaml

Comment (rahulbabu95) [reply jobs/aws/eks-anywhere-build-tooling/eks-anywhere-attribution-periodics-release-0.20.yaml:1]: I think the same file is shown in diff as renamed .?

**Review by abhay-krishna — APPROVED** [review]
/lgtm
/approve

**Pull request merged**